R/calculate.R

```diff
@@ -23,6 +23,13 @@
 #'
 #' @return A tibble containing a `stat` column of calculated statistics.
 #'
+#' @section Missing values:
```

Comment:
It might be nit-picking, but `NaN`s are not actually "missing values".
I am struggling to come up with the title that both represents the real concern (tell about this functionality) and is succinct enough. Maybe "Missing levels in small samples"? What do you think?